cmd/notary-server/dev-config.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
I have no opinions on whether we should also keep a `dev-config.json`, but I think the one being deleted uses a local trust, as opposed to a remote notary-signer service in the regular `config.json`.